content/guides/events-api-connection/warning.md

```diff
@@ -0,0 +1,18 @@
+# Limitations
+
+## No retry
+In some message brokers, the retry mechanism is natively implemented. 
+What does it mean? In brief, when a message publication failed, the system will automatically reattempt it later. 
+Very useful, isn’t it? Especially in case of temporarily network issues: the message won’t be lost.
+Unfortunately, we don’t have such a retry mechanism in our event API connection. 
+This means that if a message is not published, no mechanism will try to do it again, and the message will be forever lost...
+
+## Requests limitation
+For some performance issues, the number of requests is limited to 4000 per hour.
```

Comment:
We should probably only add a link to the existing docs https://api.akeneo.com/events-documentation/limits-and-scalability.html